src/components/AddCharacterForm/style.css

```diff
@@ -0,0 +1,28 @@
+.addCharacterForm {
+	background-color: darkcyan;
+	padding: 10px 0;
+	border: 1px solid black;
```

Comment:
be careful of tabs

Reply:
fixed on [1ef0d92](https://github.com/Svatoji/tsreact.unity/pull/1/commits/1ef0d92122ad01e0fddb4db1f8cb7e89cbe70a2b)